### ISSUE TITLE
Browse and load Lora with local thumbnail preview.

### ISSUE THREAD

**Issue by shivshankar11** [audioscavenger/ComfyUI-Thumbnails#11]
Feature request -
Browse and load Lora with local thumbnail preview just like images.
both model lora and stack lora loader will be appreciated.

**Comment by audioscavenger** [comment]
there is a node that does that, more or less by pulling images from the web. can't remember the name
where would the thumbnail come from anyway? sidecar image? and what name format for the image?

**Comment by audioscavenger** [comment]
this is what you want: https://github.com/X-T-E-R/ComfyUI-EasyCivitai-XTNodes
unfortunately its does not work for me, project seems dead. I suggest you fork and revive it

**Comment by audioscavenger** [comment]
I agree that would be awesome, the names of the checkpoints/lora are so random and cannot be sorted alphabetically. if you rename, then you cannot share your workflow anymore. 
There is another node that does that through an extra layer/portal: AIGODLIKE-ComfyUI-Studio
It looks for images of same name as the checkpoints/Lora. But will not download them. Maybe a feature I should add, I would use it myself

**Comment by HerlanderCarvalho** [comment]
> this is what you want: https://github.com/X-T-E-R/ComfyUI-EasyCivitai-XTNodes unfortunately its does not work for me, project seems dead. I suggest you fork and revive it

It works for me. Yours is not working though, the LoadImage node, now has [object] entries on the list (which I assume are for each subfolder I have inside the Input folder), but doesn't seem to work. No thumbnails of any sort, anywhere. The options show up in the settings though.

**Comment by phazei** [comment]
> unfortunately its does not work for me, project seems dead.

**Comment by audioscavenger** [comment]
not sure I want to open this crab's nest. the problem is that i made this node for LoadImage not for other schemes like load Lora or ckpt etc, I mean it could become the defacto thumbnail node loader and all, but I am lone working on this and I have a life and a family, unfortunately.

help me find a 6 figures job and then I will consider this option
